### PR TITLE
Add a state-reset script

### PR DESCRIPTION
**Pull request by zyga**
Signed-off-by: Zygmunt Krynicki zygmunt.krynicki@canonical.com

### CODE REVIEW

File: reset-state [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+set -x
+rm -rvf /var/lib/snapd/*
+set +x
+
+echo
+echo "Removing generated systemd units..."
+echo
+set -x
+rm -vf /etc/systemd/system/snap-*.mount
+rm -vf /etc/systemd/system/snap-*.service
```
Comment (mvo5) [inline reset-state:52]: We probably want to kill the symlinks in /etc/systemd/system/multi-user.target.wants/snap-*.{mount,service} here too

File: reset-state [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+set +x
+
+echo
+echo "Removing generated executable wrappers..."
+echo
+set -x
+rm -vf /snap/*
```
Comment (mvo5) [inline reset-state:60]: And _maybe_ the snap data in /var/snap, not sure about this

File: reset-state (hunk 1 above)
Comment (zyga) [inline reset-state:52]: Thanks, good point!

File: reset-state (hunk 2 above)
Comment (zyga) [inline reset-state:60]: Currently I'm not removing the state of any particular snap. Nothing needed this so far

**Pull request merged**